src/test/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocumentTest.java

```diff
@@ -611,4 +611,59 @@ public void testNoTooltip() throws Exception {
         int start = doc.getText(0, doc.getLength()).indexOf("Tj");
         Assertions.assertNull(doc.getToolTipAt(start));
     }
+
+    @Test
+    public void testIndentBaseline() throws Exception {
+        byte[] origBytes = Files.readAllBytes(Paths.get(SRC_DIR, "baseline.cmp"));
+        StyledSyntaxDocument doc = new StyledSyntaxDocument();
+        doc.processContentStream(origBytes);
+
+        String output = doc.getText(0, doc.getLength());
+        String expectedResult = new String(Files.readAllBytes(Paths.get(SRC_DIR, "baselineIndented.cmp")));
```

Comment:
@RonSkons Just a head's up that it's failing the CI quality gate at the moment due to new String(byte[]...) not having the character encoding set for this and the other 4 test cases.
If you could add that in, it should pass.

Reply:
Should be fixed now.